scripts/pythondistdeps.py

```diff
@@ -191,6 +191,11 @@
                             py_deps[name].append(spec)
                 if not dep.specs:
                     py_deps[name] = []
+            # console_scripts/gui_scripts entry points need pkg_resources from setuptools
+            if ('setuptools' not in py_deps and
+                    (dist.get_entry_map('console_scripts') or
+                        dist.get_entry_map('gui_scripts'))):
+                py_deps['setuptools'] = []
```

Comment:
shouldn't that be `python{dist.py_version}dist(setuptools)`?

Reply:
yes it should. omg

Reply:
amended